Bottom: 5343a7c8de80e208cf54f0590c0655ed3cedb7c5
Top:    8162ea9f597ebb55fe74b31f2b52308a808f4ad5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-23 02:13:03 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index a1d6bce..9e485d1 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2460,7 +2460,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
